packages/graph/deployment/migrations/Dockerfile

```
@@ -3,6 +3,13 @@ FROM node:18.7.0-alpine3.16
 COPY migrations/ /migrations/
 
 WORKDIR /migrations
+
+## Install dependencies without resolving devDependencies
+## Context: https://github.com/yarnpkg/yarn/issues/3630#issuecomment-641959126
+RUN apk add --no-cache jq && \
+  jq 'del(.devDependencies)' package.json > package.json.tmp && \
+  mv package.json.tmp package.json
+
 RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn yarn install
```

I tried 
```suggestion
RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn yarn install --production
```

but it did not work. `yarn` still tried to resolve `devDependencies` and failed to find `@local/tsconfig` in the global registry. Issue details: https://github.com/yarnpkg/yarn/issues/3630#issuecomment-641959126